src/test/confluent-platform-example/docker-compose.yml

```diff
@@ -0,0 +1,184 @@
+---
+version: '2'
+services:
+
+  broker:
+    image: confluentinc/cp-kafka:7.6.1
+    hostname: broker
+    container_name: broker
+    ports:
+      - "9092:9092"
+      - "9101:9101"
+    environment:
+      KAFKA_NODE_ID: 1
+      KAFKA_LISTENER_SECURITY_PROTOCOL_MAP: 'CONTROLLER:PLAINTEXT,PLAINTEXT:PLAINTEXT,PLAINTEXT_HOST:PLAINTEXT'
```

Comment:
Is all of this copied from somewhere and also necessary? I don't think we need to understand every line here, but assuming it's copied from somewhere, include a link to that location so if someone wants to understand what all this config is doing, they can do so.